jerry-core/ecma/builtin-objects/ecma-builtin-helpers.h

```diff
@@ -174,6 +174,20 @@ ecma_value_t
 ecma_builtin_helper_error_dispatch_call (ecma_standard_error_t error_type, const ecma_value_t *arguments_list_p,
                                          ecma_length_t arguments_list_len);
 
+/* ecma-builtin-helpers-sort.c */
+
+/**
+ * Comparison callback function header for sorting helper routines.
+ */
+typedef ecma_value_t (*sort_compare_t)(ecma_value_t lhs, /**< left value */
```

Comment:
Please use the `ecma_builtin_helper_` prefix: `ecma_builtin_helper_sort_compare_fn_t`

Reply:
@LaszloLango Done.